modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/blueprint/test/SXPBlueprintSearchResultTest.java

```diff
@@ -2716,6 +2749,7 @@ private void _reset() {
 			_assetTag = null;
 			_content = StringPool.BLANK;
 			_fieldName = StringPool.BLANK;
+			_titleMap = null;
```

Comment:
Bchan probably will change the order to alphabetical, so keep in mind that when you send your changes, put things in alphabetical order.

Reply:
@AlmirFe @felipelorenz